shared/plugins/i18n/locales/de.json

```diff
@@ -260,7 +271,7 @@
       "remember": "Eingeloggt bleiben",
       "asCommunity": "Als Community"
     },
-    "extra": "Zusätzliche infos",
+    "extra": "zusätzliche Infos",
```

Comment:
@gamersi Shouldn't this be capitalized?

Reply:
I dont know the context. if it is in a sentence zusätzliche is not capitalized. if it is standalone zusätzliche is capitalized

Reply:
@gamersi Then please refer to `en.json`. It's capitalized there.

Reply:
changed it in the newest commit.